test/index.ts

```diff
@@ -610,16 +610,14 @@ describe('.getToken()', () => {
 });
 
 function createGetTokenMock(code = 200, body?: {}) {
-  return nock(GOOGLE_TOKEN_URLS[0])
+  return nock(GOOGLE_TOKEN_URLS[0], {
```

Comment:
This makes me sad, and scared about any header assertions done elsewhere in our codebases